### ISSUE TITLE
model下载不了

### ISSUE THREAD

**Issue by Puersip** [bengxy/FastNeuralStyle#4]
给出的链接不行了
能再放一个链接吗，谢谢

**Comment by niguanga** [comment]
是下不了地

**Comment by bengxy** [comment]
年代太久远，当时的网盘文件已经过期了，而且pytorch经过这么多年，版本也发生了很大变化

只有模型结构是值得参考的，我记得当时实现最大的坑在loss函数上面，其他地方都很简单

The pre-trained model checkpoint is out of date, and after a few years, pytorch version has changed a lot

You can still ref to the model structure. The most IMPORTANT part is the design of the loss function. The other parts are very simple.
